src/types/courierDeliveryDetailsValues.ts

```diff
@@ -0,0 +1,12 @@
+export type CourierDeliveryDetailsInitialValues = Record<string, never>;
```

Comment:
You don't need it because a type for form value screen provides the same defaults